power_grid_model_c/power_grid_model/include/power_grid_model/component/transformer.hpp

```diff
@@ -71,14 +73,14 @@ class Transformer : public Branch {
     double phase_shift() const final { return clock_ * deg_30; }
     bool is_param_mutable() const final { return true; }
     // getters
-    constexpr IntS tap_pos() const { return tap_pos_; }
-    constexpr BranchSide tap_side() const { return tap_side_; }
-    constexpr IntS tap_min() const { return tap_min_; }
-    constexpr IntS tap_max() const { return tap_max_; }
-    constexpr IntS tap_nom() const { return tap_nom_; }
+    IntS tap_pos() const { return tap_pos_; }
+    BranchSide tap_side() const { return tap_side_; }
+    IntS tap_min() const { return tap_min_; }
+    IntS tap_max() const { return tap_max_; }
+    IntS tap_nom() const { return tap_nom_; }
```

Comment:
why are these not `constexpr` anymore? there's no complicated logic in the function that requires it to be non-`constexpr` and it may help the compiler

Reply:
This is weird, I didn't do that; guess the formatter updated this -> I'll dive into it